configure.ac

```diff
@@ -7,7 +7,7 @@ m4_define([PKG_VERSION], m4_bpatsubst(m4_esyscmd([grep "Version:" firewalld.spec
 m4_define([PKG_RELEASE], m4_bpatsubst(m4_esyscmd([grep "Release:" firewalld.spec]), [Release:\W\([0-9.]*\).*\W], [\1]))
 m4_define([PKG_TAG], m4_format(v%s, PKG_VERSION))
 
-AC_INIT(PKG_NAME,PKG_VERSION)
+AC_INIT([PKG_NAME],[PKG_VERSION])
```

Comment:
autoreconf or one of the other auto* commands made this change when I ran them & it looks correct.